src/_redirects

```diff
@@ -1,4 +1 @@
 # Redirects from what the browser requests to what we serve
-/posts/2022/ruby-hash-dot-syntax-performance-benchmarks       /posts/2022/ruby-hash-dot-syntax-dig-performance-benchmarks
-
-/posts/2022/ruby-hash-dot-syntax-dig-performance-benchmarks   /posts/2022/ruby-hash-dot-syntax-deep-fetch
```

Comment:
(Unrelated change; I noticed while reviewing the files generated in the Bridgetown deployment output that these were left over from the blog from which I copied the template for this blog.)